lib/string_reverse.rb

```diff
@@ -1,4 +1,21 @@
 # A method to reverse a string in place.
 def string_reverse(my_string)
-  my_string << "not implemented"
+  # my_string << "not implemented"
+
+  if my_string == nil
+    return my_string
+  else
+    i = 0
+    j = my_string.length - 1
+
+    while i < j
+      temp = my_string[i]
+      my_string[i] = my_string[j]
+      my_string[j] = temp
+
+      i += 1
+      j -= 1
+    end
+  end
+  return my_string

```

Comment:
Since this method reverses the string in-place, we don't need to return my_string. You could change lines 6 and 20 to simply be `return`.